### PR TITLE
Scipy blog

### PR DESCRIPTION
**Pull request by dgasmith**
Adds a SciPy2019 blog post.

I also nuke the featured image in the blog post... I think they were a bit ugly and far too big.

### CODE REVIEW

File: themes/small-apps/assets/scss/pages/_blog.scss [hunk 1]
```diff
@@ -95,9 +95,12 @@
 		.post-body {
 			.feature-image {
 				margin-bottom: 30px;
+                margin-left: 40%;
+                margin-right: auto;
+                width: 40%;
 
 				img {
-					width: 100%;
+					height: 75px;
```
Comment (Lnaden) [inline themes/small-apps/assets/scss/pages/_blog.scss]: Do we only want the pictures to be 75px high?

File: content/blog/2019-07-30-SciPy20109.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+covered some of the capabilities of the QCArchive ecosystem and projects that
+use this ecosystem.
+
+<center>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/HBkY_qnYSQw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</center>
+
+-------------
+
+It was fantastic to see friends that are part of many academic groups,
```
Comment (Lnaden) [inline content/blog/2019-07-30-SciPy20109.md]: Is it possible to link out to all of these groups? Should we?

File: themes/small-apps/assets/scss/pages/_blog.scss (hunk 1 above)
Comment (dgasmith) [reply themes/small-apps/assets/scss/pages/_blog.scss]: It was an attempt to have the feature images be somewhat reasonable. This didn't work out so I simply removed them, I think it looks a lot better and is much more readable.

File: content/blog/2019-07-30-SciPy20109.md (hunk 2 above)
Comment (dgasmith) [reply content/blog/2019-07-30-SciPy20109.md]: I attempted to at the beginning, but eventually decided against it.

**Pull request merged**